includes/class-activity-dispatcher.php

```diff
@@ -31,7 +31,8 @@ public static function send_post_activity( $activitypub_post ) {
 		$activitypub_activity->from_post( $activitypub_post->to_array() );
 
 		foreach ( \Activitypub\get_follower_inboxes( $user_id ) as $inbox => $to ) {
-			$activitypub_activity->set_to( $to );
+			$activitypub_activity->set_to( 'https://www.w3.org/ns/activitystreams#Public' );
```

Comment:
Does it have to be a string? if we only set the `cc`, `to` falls back to the default and that is https://github.com/pfefferle/wordpress-activitypub/blob/master/includes/model/class-activity.php#L17